srcpkgs/ibus-chewing/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-chewing'
+pkgname=ibus-chewing
+version=1.5.1
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="git cmake pkg-config"
+makedepends="gettext-devel libchewing-devel ibus-devel gob2 libX11-devel gtk+3-devel glib-devel findutils"
+depends="ibus libchewing"
+short_desc="Chewing input method for ibus"
+maintainer="ctoid <funk443@yandex.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/definite/ibus-chewing"
+
+do_fetch() {
+	git clone https://github.com/definite/ibus-chewing.git $wrksrc
+}
+
+pre_configure() {
+	rm -rf cmake-fedora
+	git clone https://pagure.io/cmake-fedora.git
+}
+
+do_configure() {
+	cmake . -DCMAKE_INSTALL_PREFIX='/usr' -DLIBEXEC_DIR='/usr/libexec'
```

Comment:
For this, you shouldn't need to add a custom do_configure, you should be able to just do `configure_args`.

Reply:
and `-DCMAKE_INSTALL_PREFIX='/usr'` is already defined in the build-style so it doesn't need to specified.